gridappsd/goss.py

```diff
@@ -212,7 +219,14 @@ def _make_connection(self):
             self._conn = Connection([(self.stomp_address, self.stomp_port)])
             if self._override_thread_fc is not None:
                 self._conn.transport.override_threading(self._override_thread_fc)
-            self._conn.connect(self.__user, self.__pass)
+            try:
```

Comment:
This try/except block worries me, as it seems to leave ample opportunities to fail silently.